### PR TITLE
agent keepalive support

### PR DESCRIPTION
**Pull request by rocketbitz**
Cleans up the changes in https://github.com/drone/autoscaler/pull/15. Trying to run this myself, but experiencing issues on GCP, and would like to get this merged in, and not seeing any movement on https://github.com/drone/autoscaler/pull/15.

### CODE REVIEW

File: engine/install.go [hunk 1]
```diff
@@ -12,19 +12,20 @@ import (
 	"sync"
 	"time"
 
-	"github.com/drone/autoscaler"
```
Comment (tboerger) [inline engine/install.go]: You should keep the import like it was, it's don't like that intentional ;)

**Comment by genisd** [comment]
Thank you @rocketbitz 
I'm sorry I couldn't really work on it anymore since we had some higher priority stuff going on.
So many thanks for finishing this 👍

**Comment by genisd** [comment]
Thank you @rocketbitz 
I'm sorry I couldn't really work on it anymore since we had some higher priority stuff going on.
So many thanks for finishing this 👍

**Comment by genisd** [comment]
Thank you @rocketbitz 
I'm sorry I couldn't really work on it anymore since we had some higher priority stuff going on.
So many thanks for finishing this 👍

**Comment by yellowmegaman** [comment]
So it won't be in 0.8 release? The point is that 1.0 autoscaler is incompatible (for all i know) with drone server 0.8, so everyone on 0.8 are still suffering from this bug.

**Comment by bradrydzewski** [comment]
@yellowmegaman not sure what you mean ... this pull request was merged and there is a `drone/autoscaler:0.8.0` snapshot in dockerhub that you can download.

**Pull request merged**